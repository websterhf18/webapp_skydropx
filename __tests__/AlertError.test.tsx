/**
 * @format
 * @jest-environment jsdom
 */
import React from "react";
import renderer from "react-test-renderer"; // Note: test renderer must be required after react-native.
import LabelErrorAlert from "../components/Alerts/LabelError.alert";

import { Provider } from "react-redux";
import store from "../models/index";

it("Renders the Label Error Alert Component", () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <LabelErrorAlert showAlert={true} />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
