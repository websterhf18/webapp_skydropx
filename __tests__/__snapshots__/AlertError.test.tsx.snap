// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Label Error Alert Component 1`] = `
<div
  className="my-alert-top alert alert-danger alert-dismissible d-block"
>
  <strong>
    Error!
  </strong>
   No se puede abrir el documento
  <button
    className="btn-close"
    onClick={[Function]}
    type="button"
  />
</div>
`;
